conf/ocsci/openshift_dedicated.yaml

```diff
@@ -10,8 +10,10 @@ ENV_DATA:
   skip_ocs_deployment: true
 
 # AUTH:
-#  openshiftdedicated:
-#    token: OCM TOKEN KEY PLACEHOLDER
+#   pagerduty:
+#     api_key: PAGERDUTY API KEY PLACEHOLDER
```

Comment:
I would stress that this pagerduty key is ocp dedicated only (to make it clear for others who may read this and will wonder whether they need to handle this or not).